dev/column.h

```diff
@@ -109,6 +109,48 @@ namespace sqlite_orm {
             }
         };
 
+        // we are compelled to wrap all sfinae-implemented traits to prevent "error: type/value mismatch at argument 2 in template parameter list"
+        namespace sfinae {
+            /**
+             *  Column with insertable primary key traits. Common case.
+             */
+            template<class T, class SFINAE = void>
+            struct is_column_with_insertable_primary_key : public std::false_type {};
+
+            /**
+             *  Column with insertable primary key traits. Specialized case case.
+             */
+            template<class O, class T, class... Op>
+            struct is_column_with_insertable_primary_key<
+                column_t<O, T, Op...>,
+                typename std::enable_if<(tuple_helper::tuple_contains_type<
+                                         constraints::primary_key_t<>,
+                                         typename column_t<O, T, Op...>::constraints_type>::value)>::type> {
+                using column_type = column_t<O, T, Op...>;
+                static constexpr bool value = is_primary_key_insertable<column_type>::value;
+            };
+
+            /**
+             *  Column with noninsertable primary key traits. Common case.
+             */
+            template<class T, class SFINAE = void>
+            struct is_column_with_noninsertable_primary_key : public std::false_type {};
+
+            /**
+             *  Column with noninsertable primary key traits. Specialized case case.
+             */
+            template<class O, class T, class... Op>
+            struct is_column_with_noninsertable_primary_key<
```

Comment:
or you can make only one common specification and use `value = !is_column_with_insertable_primary_key`

Reply:
No, it will break the logic

Reply:
```
static_assert(
    count_tuple<columns_type, is_column_with_insertable_primary_key>::value <= 1,
    "Attempting to execute 'insert' request into an noninsertable table was detected. "
    "Insertable table cannot contain > 1 primary keys. Please use 'replace' instead of "
    "'insert', or you can use 'insert' with explicit column listing.");
static_assert(
    count_tuple<columns_type, is_column_with_noninsertable_primary_key>::value == 0,
    "Attempting to execute 'insert' request into an noninsertable table was detected. "
    "Insertable table cannot contain non-standard primary keys. Please use 'replace' instead "
    "of 'insert', or you can use 'insert' with explicit column listing.");
```
Second static_assert will be triggered for all normal columns, even if column is not a primary key